addons/website_blog/models/website_blog.py

```diff
@@ -21,6 +21,13 @@ class Blog(models.Model):
     subtitle = fields.Char('Blog Subtitle', translate=True)
     active = fields.Boolean('Active', default=True)
     content = fields.Html('Content', translate=html_translate, sanitize=False)
+    blog_post_ids = fields.One2many('blog.post', 'blog_id', 'Blog Posts')
+    blog_post_count = fields.Integer("Posts", compute='_compute_blog_post_count', store=True)
```

Comment:
@JKE-be Opinion on this: do we need to store it?

Reply:
non

Comme tout a l'heure, c une vue qui n'est jamais utilisée... 
Il n'y a d'interet a storer seulement quand le calcul est couteux et/ou qu'il est affiché souvent.

Reply:
J'avais en effet oublié de corriger cette ligne ! Je change ça desuite !